### DISCUSSION TITLE
Change Host Name in the header (running in a docker container)

### DISCUSSION THREAD

**Discussion by janreinhardt** [nicolargo/glances#2863] [Q&A] [answered]
Hi,
I have read the manuals and asked google, but I cannot find a way to change my machines (random?) host name which is displayed in the header line of my Glances, running as a Docker container = webserver mode:
`f3a8bef022b0 Debian GNU/Linux 12 64bit / Linux 6.6.31+rpt-rpi-2712`

This looks kinda ugly, how do I change the f3a… stuff?
Thanks!

**Comment by nicolargo** [comment, marked as answer]
Hi @janreinhardt 

how do you run Glances ?

With the following command line:

    docker run --rm -e TZ="${TZ}" -v /var/run/docker.sock:/var/run/docker.sock:ro -v /run/user/1000/podman/podman.sock:/run/user/1000/podman/podman.sock:ro --pid host --network host -it nicolargo/glances:latest

The UI display the hostname:

![image](https://github.com/nicolargo/glances/assets/776747/24038110-2202-4339-8e05-e11386146f96)

You can also overwrite the message using the Glances configuration file:

```
[system]
# System information to display (a string where {key} will be replaced by the value)
# Available information are: hostname, os_name, os_version, os_arch, linux_distro, platform
system_info_msg= | My {os_name} system |

```

**Reply by janreinhardt** [reply]
Thanks! `--network host` was missing in my docker run. Now that I added it, my hostname is shown properly.
(when in a Docker bridge network, I could have set the container‘s hostname via `--hostname FANTASY` which I just learnt but didnt know yesterday)

Happy now, so thank you again!
